apps/app/src/App.tsx

```diff
@@ -24,16 +24,15 @@ import { Prescription } from './views/routes/Prescription';
 import { PrescriptionForm } from './views/routes/PrescriptionForm';
 import { Prescriptions } from './views/routes/Prescriptions';
 import { Settings } from './views/routes/Settings';
-import { ClinicalApiProvider } from './views/routes/Settings/apollo';
 import { Support } from './views/routes/Support';
 import { UpdatePatientForm } from './views/routes/UpdatePatientForm';
+import { Env } from 'packages/sdk/src/utils';
```

Comment:
this should be imported from `@photonhealth/sdk`

Reply:
fixed